base/libgit2/libgit2.jl

```diff
@@ -215,6 +215,22 @@ end
 
 Set the `url` for `remote` for the git repository `repo`.
 The default name of the remote is `"origin"`.
+
+# Examples
+
+```julia
+julia> repo_path = joinpath("test_directory", "Example")
```

Comment:
It could also be written in "script form," i.e. without the `julia>` prefix that suggests it's being used from the REPL.